### PR TITLE
chore: rename ApproveOracleRegistration to reuse

### PR DESCRIPTION
**Pull request by H4NLee**
I renamed the `ApproveOracleRegistration` to `ApprovalSharingOracleKey` to reuse in `MsgApproveOracleUpgrade`.

### CODE REVIEW

**Review by audtlr24 — APPROVED** [review]
lgtm

**Review by gyuguen — APPROVED** [review]
LGTM

File: proto/panacea/oracle/v2/tx.proto [hunk 1]
```diff
@@ -48,12 +48,12 @@ message MsgRegisterOracleResponse {
 
 // MsgApproveOracleRegistration defines the Msg/ApproveOracleRegistration
 message MsgApproveOracleRegistration {
-  ApproveOracleRegistration approve_oracle_registration = 1;
+  ApprovalSharingOracleKey approval_sharing_oracle_key = 1;
```
Comment (inchori) [inline proto/panacea/oracle/v2/tx.proto:51]: How about renaming like `SharingOracleKeyApproval`?

File: x/oracle/keeper/oracle_test.go [hunk 2]
```diff
@@ -264,7 +264,7 @@ func (suite *oracleTestSuite) TestApproveOracleRegistrationSuccess() {
 	encryptedOraclePrivKey, err := btcec.Encrypt(suite.nodePubKey, suite.oraclePrivKey.Serialize())
 	suite.Require().NoError(err)
 
-	approveOracleRegistration := &types.ApproveOracleRegistration{
+	approveOracleRegistration := &types.ApprovalSharingOracleKey{
```
Comment (inchori) [inline x/oracle/keeper/oracle_test.go:267]: I think that variable name also changes to `approvalSharingOracleKey`.

**Review by inchori — APPROVED** [review]
LGTM!
I just left not an important question and comment.

File: proto/panacea/oracle/v2/tx.proto (hunk 1 above)
Comment (H4NLee) [reply proto/panacea/oracle/v2/tx.proto:51]: Or.. `MsgEncryptedOraclePrivKey`..?

File: x/oracle/keeper/oracle_test.go (hunk 2 above)
Comment (H4NLee) [reply x/oracle/keeper/oracle_test.go:267]: thank you I'll change it!

File: proto/panacea/oracle/v2/tx.proto (hunk 1 above)
Comment (inchori) [reply proto/panacea/oracle/v2/tx.proto:51]: Hmm.... I think that the `ApprovalSharingOracleKey` is more better. 😂

**Pull request merged**